content/en/docs/components/hyperparameter-tuning/experiment.md

```diff
@@ -142,7 +142,8 @@ These are the fields in the experiment configuration spec:
 * **resumePolicy**: Experiment resume policy. Can be one of `LongRunning`, `Never` or `FromVolume`.
   Default value is `LongRunning`.
   See the [`ResumePolicy` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_types.go#L54).
```

Comment:
```suggestion
  (Refer to the [`ResumePolicy` type](https://github.com/kubeflow/katib/blob/master/pkg/apis/controller/experiments/v1beta1/experiment_types.go#L54).)
```

Reply:
For other GO files in the doc, can you please apply this logic, if it makes sense to you? It's mainly for a11y, as well as letting the reader know why you should "see" the file, since no reasons are originally provided. I think it can be good practice.